doc/dev-guide.md

```diff
@@ -1,28 +1,29 @@
 # Stripes CLI Developer's Guide
 
-* [Introduction](#introduction)
-* [Development installation](#development-installation)
-* [Running tests](#running-tests)
-* [Code organization](#code-organization)
-* [Commands](#commands)
-    * [Options](#options)
-    * [Positionals](#positionals)
-    * [Middleware](#middleware)
-        * [CLI context](#cli-context)
-        * [Stripes config](#stripes-config)
-        * [Standard input](#standard-input)
-        * [Interactive input](#interactive-input)
-    * [Grouping](#grouping)
-* [Logging](#logging)
-* [Okapi Client](#okapi-client)
-* [Plugins](#plugins)
-* [Documentation](#documentation)
-    * [Generating the command reference](#generating-the-command-reference)
-    * [Table of Contents](#table-of-contents)
-* [Debugging](#debugging)
-    * [Visual Studio Code](#visual-studio-code)
-    * [Adding breakpoints in Stripes-core](#adding-breakpoints-in-stripes-core)
-* [Releasing](#releasing)
+- [Stripes CLI Developer's Guide](#stripes-cli-developers-guide)
```

Comment:
I take it you're using a different TOC generator than [the one described here](https://github.com/folio-org/stripes-cli/blob/master/doc/dev-guide.md#table-of-contents).  I'm not opposed to the difference in `*` and `-` syntax, however, including the root heading is redundant and adds an unnecessary level of nesting to the TOC.  If using md2toc, the `-l 2` option will eliminate that.

Reply:
Might have been caused by the Markdown plugin I'm using in VSCode. I'll re-run with md2toc to correct that.